src/test/java/com/puppycrawl/tools/checkstyle/checks/annotation/MissingOverrideCheckTest.java

```diff
@@ -227,6 +228,61 @@ public void testGoodAnnotationOverrideJ5Compatible() throws Exception {
                 getPath("InputMissingOverrideGoodOverrideJava5.java"), expected);
     }
 
+    /**
+     * This tests that record accessor methods without @Override are correctly flagged.
+     */
```

Comment:
remove all comments and make sure method name is named with all such words.
Long name of test is completely ok.

if you have no good ideas, there is rule of thumb
`This tests that record accessor methods without @Override are correctly flagged.`
->
`testThatRecordAccessorMethodsWithoutOverrideAreCorrectlyFlagged.`